app/src/main/java/com/team22/soundary/feature/share/ShareBottomSheet.kt

```diff
@@ -0,0 +1,49 @@
+package com.team22.soundary.feature.share
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import com.team22.soundary.R
+import com.team22.soundary.feature.share.data.FriendItemEntity
+
+class ShareBottomSheet : BottomSheetDialogFragment() {
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        super.onCreateView(inflater, container, savedInstanceState)
+        return inflater.inflate(R.layout.bottom_sheet, container, false)
+    }
```

Comment:
p5; 사소하지만, 이렇게 사용하면 onCreateView 를 제거할 수 있습니다.

```suggestion
class ShareBottomSheet : BottomSheetDialogFragment(R.layout.bottom_sheet)
```

Reply:
피드백대로 수정하였습니다! 불필요한 코드를 제거하여 훨씬 깔끔해진 것 같아요! 감사합니다
[0a21203](https://github.com/kakao-tech-campus-2nd-step3/Team22_Android/pull/28/commits/0a21203768d7bc9b11365dbad7f92ea1aed4e5ae)